src/Types.jl

```diff
@@ -1051,13 +1066,11 @@ function registered_info(env::EnvCache, uuid::UUID, key::String)
     isempty(paths) && cmderror("`$uuid` is not registered")
     values = []
     for path in paths
-        info = parse_toml(paths[1], "package.toml")
+        info = parse_toml(path, "package.toml")
         value = get(info, key, nothing)
-        value in values || push!(values, value)
+        push!(values, (path,value)) 
```

Comment:
`(path, value)` to match the surrounding style in this project.

Reply:
Done